src/main/java/org/apache/mesos/specification/DefaultService.java

```diff
@@ -15,59 +15,72 @@
 import java.util.Optional;
 
 /**
- * This class is a default implementation of the Service interface.  It serves mainly as an example with hard-coded
- * values for "user" and "master-uri", and failover timeouts.  More sophisticated services may want to implement the
- * Service interface directly.
+ * This class is a default implementation of the Service interface.  It serves mainly as an example
+ * with hard-coded values for "user", and "master-uri", and failover timeouts.  More sophisticated
+ * services may want to implement the Service interface directly.
+ *
+ * Customizing the runtime user for individual tasks may be accomplished by customizing the 'user'
+ * field on CommandInfos returned by {@link TaskTypeSpecification#getCommand(int)}.
  */
 public class DefaultService implements Service {
     private static final int TWO_WEEK_SEC = 2 * 7 * 24 * 60 * 60;
     private static final String MASTER_URI = "zk://master.mesos:2181/mesos";
     private static final String USER = "root";
-    private final Logger logger = LoggerFactory.getLogger(getClass());
+    private static final Logger LOGGER = LoggerFactory.getLogger(DefaultService.class);
     private final int apiPort;
     private StateStore stateStore;
     private ServiceSpecification serviceSpecification;
 
+    /**
+     * Creates and registers the service with Mesos, while starting a Jetty HTTP API service on the
```

Comment:
I think you accidentally copy/pasted comments.  The constructor doesn't `Creates and registers etc.`